ui/src/app/App.tsx

```diff
@@ -94,16 +94,28 @@ export const App = (): JSX.Element => {
   }, [dispatch, connected]);
 
   useEffect(() => {
-    if (!skipIntro && configLoaded) {
+    if (
+      !skipIntro &&
+      configLoaded &&
+      !location.pathname.startsWith(introURLs.index)
```

Comment:
Because this uses `startsWith` you could load `/MAAS/r/intro/user` with no way to get to the main intro (other than manually changing the url). I think we should add a check to UserIntro to redirect back to the main intro if it hasn't been completed. I'll add an issue.